### PR TITLE
feat: authenication page added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Authentication pages are added.
1. Login Page.
2. Sign-up page.
3. forgot the password page.
please review the code and provide feedback
Screenshot 2022-02-17 165058
Screenshot 2022-02-17 165040
Screenshot 2022-02-17 165017

### CODE REVIEW

File: authentication/forgot_password/forgotPass.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+
```
Comment (Kothalkarpriya) [inline authentication/forgot_password/forgotPass.css]: remove these extra space

File: authentication/forgot_password/forgotPass.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <link rel="stylesheet" href="/cart_page/cart_page.css">
+    <link rel="stylesheet" href="/authentication/login_page/login.css">
+    <link rel="stylesheet" href="/authentication/forgot_password/forgotPass.css">
+</head>
+<body class="body-container">
+    <nav class="navbar">
+        <h3 class="nav__heading">Lorem</h3>
+        <div class="search-container">
+            <input type="search" name="searchBar" class="input__search" placeholder="search here">
+            <a href="#"><button class="btn__search"><img src="/images/home-page/search.png" alt="search" class="img--icon"></button></a>
```
Comment (Kothalkarpriya) [inline authentication/forgot_password/forgotPass.html]: for closing image tag do include / before > tag

File: authentication/forgot_password/forgotPass.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        </div>
+        <div class="customer-info">
+           <a href="/authentication/login_page/login.html" class="link"> <button class="btn--login">Login</button> </a>
+           <a href="/wishlist/wishlist.html" class="item--link"> <div class="wishlist-box">
+                <div class="badge">
+                    <img src="/images/home-page/wishlist.png" alt="wishlist image" class="badge__img">
```
Comment (Kothalkarpriya) [inline authentication/forgot_password/forgotPass.html]: here also

**Review by Kothalkarpriya — COMMENTED** [review]
The code looks good, but look into my comment for the image tag. In the deploy version you haven't linked authentication page can't able to see how it looks like

File: authentication/signUp_page/signUp.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    flex-wrap: wrap;
+    justify-content: center;
+    border: none;
+    row-gap: 1rem;
+    border: 2px solid var(--primary-color);
+    padding: 4rem;
+}
+
+
+
```
Comment (mariyasada) [inline authentication/signUp_page/signUp.css]: remove unnecessary space

**Review by mariyasada — APPROVED** [review]
you can merge it

File: authentication/forgot_password/forgotPass.css (hunk 1 above)
Comment (ShreyashMahajan) [reply authentication/forgot_password/forgotPass.css]: ok

File: authentication/forgot_password/forgotPass.html (hunk 2 above)
Comment (ShreyashMahajan) [reply authentication/forgot_password/forgotPass.html]: ok

File: authentication/forgot_password/forgotPass.html (hunk 3 above)
Comment (ShreyashMahajan) [reply authentication/forgot_password/forgotPass.html]: ok

**Comment by ShreyashMahajan** [comment]
> The code looks good, but look into my comment for the image tag. In the deploy version you haven't linked authentication page can't able to see how it looks like

ok, thank you , I will look into that

File: authentication/signUp_page/signUp.css (hunk 4 above)
Comment (ShreyashMahajan) [reply authentication/signUp_page/signUp.css]: ok

**Comment by ShreyashMahajan** [comment]
> you can merge it

thank you

**Review by Kothalkarpriya — APPROVED** [review]

**Pull request merged**